### ISSUE TITLE
[BUG] like #1951

### ISSUE THREAD

**Issue by rubensutter** [Leantime/leantime#2101]
**What set up are you using**
* [ ] Cloud Hosted Version
* [ X ] Self Hosted

**Describe the bug**

According to [BUG] Jibberish inside of Your To-Dos Title on main Page #1951
the german translation file de-DE.ini seems to contain additional expressions like

subtitles.todos_this_week="Aufgaben für diese Woche (<span class='thisWeekCounter'>%1$s</span>)"
in comparison with the version in the file en-US.ini containing just
subtitles.todos_this_week = "To-Dos this Week"

This results in showing "Aufgaben für diese Woche (%1$s) ({x})" and "anstehende Aufgaben (%1$s) ({x})"
with {x} showing the correct number of todos.

**To Reproduce**

Set the language to 'de' (Deutsch)
Navigate to the dashboard

**Expected behavior**

Aufgaben für diese Woche ({x})
anstehende Aufgaben ({x})
showing up in the Dashboards To-Dos module

**Leantime Version**
Which Leantime version are you using

2.4.7

**Server**
Apache

**PHP / MySQL Version**
Please list the PHP and Mysql version you are using
8.1.2 | 8.1.25
mysql Ver 8.0.35 | mariadb

**Additional context**

**Comment by jontorrado** [comment]
You are right. I've check some of the translations files, here are some results. When finding occurreces of "%1$s":
* en-US (most updated file): 56 times.
* en-GB: 58 times
* es-ES: 59 times
* de-DE: 58 times
* fr-FR: 57 times

This requires some time to debug which ones deserve that string and which ones don't, and then update 35 languages (all except for en-US).
